data/liveinst/liveinst

```diff
@@ -33,10 +33,6 @@ if [[ "$LIVECMD $*" =~ "--liveinst" ]]; then
     LIVE_INSTALL=1
 fi
 
-# Start swap on ZRAM service. Do this for LiveOS only, rather than enabling
-# it on the target system, for the time being.
-systemctl start zram.service
```

Comment:
Can we drop this?

Reply:
If Anaconda won't depend on this service than it should be handled by Lorax/Live creation.